.github/workflows/push-master.yml

```diff
@@ -36,3 +36,18 @@ jobs:
                        --tag=latest
       env:
         PROJECT_ID: ${{ secrets.PROJECT_ID }}
+    - name: Push Load Generator Image to GCR
+      timeout-minutes: 20
+      run: |
+        set -x
+        gcloud auth configure-docker --quiet
+        # build and tag with git hash
+        IMAGE_SHA=gcr.io/$PROJECT_ID/sandbox/loadgenerator:$GITHUB_SHA
+        docker build -t $IMAGE_SHA ./src/loadgenerator
```

Comment:
Should we have a different path here to reflect new loadgen changes compared to previous image?

Reply:
oop! let me fix this up, slipped my mind